src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    Ball getRandomBall() {
+        Ball ball = new Ball();
+        ball.color = String.valueOf(new ColorSupplier().getRandomColor());

```

Comment:
You are creating a new `ColorSupplier` instance each time `getRandomBall()` is called. Consider creating a single instance of `ColorSupplier` as a class member to avoid unnecessary object creation.